projects/transformers/callbacks/metrics.py

```diff
@@ -40,6 +41,7 @@ def __init__(self):
             self.eval_metrics['acc'] -> [acc1, acc2, ..., accn]
             self.steps = [eval_steps, eval_steps*2, ..., eval_steps*n]
         """
+        self.sparsity_tolerance = sparsity_tol if sparsity_tol is not None else 0.01
```

Comment:
I agree the 80-chars line doesn't help at all in using good variable names. But in this case I think it is easier in general if we can make the name of the argument and class attribute matches - it is always confusing when they don't